src/client/protocolgamesend.cpp

```diff
@@ -934,11 +934,11 @@ void ProtocolGame::sendSeekInContainer(int cid, int index)
     send(msg);
 }
 
-void ProtocolGame::sendInspectionNormalObject(const Position& position)
+void ProtocolGame::sendInspectionNormalObject(const Otc::InspectObjectTypes inspectionType ,const Position& position)
```

Comment:
In this situation it would only be INSPECT_NORMALOBJECT, do you think you need to send the InspectionType as a parameter?

Reply:
modify it, you know more about cpp than me. i just fixed a compiling error and i think it didn't work, when i execute the command in console, the information is not sent to the server.

Reply:
I commented because I had done it correctly and you modified it.